_includes/head.html

```diff
@@ -124,6 +124,9 @@
     <script>
         var _paq = window._paq = window._paq || [];
         /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+        {%- unless site.matomo_cookies %}
```

Comment:
```suggestion
        {%- if site.matomo_cookies == false %}
```

I prefer it reverted. So if people don't want cookies, they have to explicitly say false.

Reply:
I think these two statements are equivalent, no? But perhaps the more explicit `if` version is better indeed.

Reply:
Applied suggestion in b05d71f.

Reply:
With you unless statement you do not capture when the variable is not specified, making it false, instead of the desired true.